auth-web/src/components/auth/Transactions.vue

```diff
@@ -342,6 +351,15 @@ export default class Transactions extends Vue {
     this.updateTransactionTableCounter++
     this.setFilterArray()
   }
+
+  private async exportCSV () {
```

Comment:
Transaction has got month, week filters as well right? Aren't we doing it now?

Reply:
Yes, it has last month and last week filters.
export will send whatever filter currently applied in the list.

Reply:
ohh ok, so its not using the weekFilter and monthFilter in API. It will be always dateFilter.